.github/workflows/acceptance.yml

```diff
@@ -108,11 +108,14 @@ jobs:
           save-only: true
           snapshot-path: .artifacts/visual-snapshots
 
+      # This job runs when FE or BE changes happen.
+      # In order to prevent codecov confusion because we use the carryforward flag we should
+      # post to a different codecov flag
       - name: Handle artifacts
         uses: ./.github/actions/artifacts
         with:
           files: .artifacts/coverage/*
-          type: frontend
+          type: acceptance
```

Comment:
Possibly bad idea, but would naming this something like codecov-frontend, codecov-* make this a little easier to follow?

My original response to this PR was that the name didn't match the job name and so I thought it was wrong, but I see (or at least think) that this is mapping to paths in codecov.yml.

Maybe `codecov-fe-only`, `codecov-be-only` and `codecov-fe-and-be` would be appropriate?

Reply:
If you're considering a flag rename, here's the validation regex from [our docs](https://docs.codecov.com/docs/flags#one-to-one-relationship-of-flags-to-uploads):

`^[\w\.\-]{1,45}$`

All flag names must be valid by that regex, and I believe all the above suggestions are.

Reply:
@mattgauntseo-sentry what do you think of naming this one `backend-and-frontend`? I would rather not rename all of them (prevent confusion with those that are used to the frontend and backend flags).

@eliatcodecov It is probably important to detect this issue at the codecov level and inform the users. The confidence we had on codecov was very low because we were unable to understand why everything was so flaky.

Reply:
@armenzg understood. The tricky problem to solve here is that there are legitimate use cases for uploading to the same flag multiple times, so just on its face this particular yaml configuration isn't necessarily an error. 

Your point still stands, though. I'll discuss with the team on this. At the very least we may be able to deliver a warning to the effect of "hey, are you sure you want to be doing this?"

Reply:
@armenzg `backend-and-frontend` works for me. 

The rationale for the prefix is it's difficult at the moment to reason with which values in github actions are used by other services, for example the action names that are used by eng-pipes/freight for status checks.